### DISCUSSION TITLE
[SQL Lab] Double click to edit tab title

### DISCUSSION THREAD

**Discussion by ktmud** [apache/superset#18425] [Ideas]
**Is your feature request related to a problem? Please describe.**

Currently you have to click three times in order to edit the title of a SQL Lab tab:

<img src="https://user-images.githubusercontent.com/335541/102665507-c80a6d80-4139-11eb-9779-1a5e25349a17.png" 
 width="400" />

Considering the tab title is almost the most important thing you want to change on a tab, it would be nice if we can make editing it easier. 

**Describe the solution you'd like**

- Click on a focused tab enters edit mode
   <img src="https://user-images.githubusercontent.com/335541/102665802-57b01c00-413a-11eb-8bdd-848eb6e305d0.png" width="400">
- Blurring from the input saves the title

**Describe alternatives you've considered**

**Additional context**

**Comment by yousoph** [comment]
Thanks for the suggestion!! Will look into an improvement as part of some upcoming UI changes that we're working on :)

**Comment by pranav1610** [comment]
I would love to work on this issue. Is it available to assign?

**Comment by yousoph** [comment]
Yes, @pranav1610 , feel free to work on this!
